vulnerabilities/importers/redhat.py

```diff
@@ -91,19 +91,24 @@ def to_advisory(advisory_data):
     if bugzilla:
         url = "https://bugzilla.redhat.com/show_bug.cgi?id={}".format(bugzilla)
         bugzilla_data = requests.get(f"https://bugzilla.redhat.com/rest/bug/{bugzilla}").json()
-        bugzilla_severity_val = bugzilla_data["bugs"][0]["severity"]
-        bugzilla_severity = VulnerabilitySeverity(
-            system=scoring_systems["rhbs"],
-            value=bugzilla_severity_val,
-        )
+        if (
+            ("bugs" in bugzilla_data)
+            and len(bugzilla_data["bugs"])
+            and ("severity" in bugzilla_data["bugs"][0])
```

Comment:
this could be made more concise by using `bugzilla_data.get["bugs"][0].get("severity")`